ql/instruments/bonds/fixedratebond.cpp

```diff
@@ -120,6 +125,7 @@ namespace QuantLib {
         cashflows_ = FixedRateLeg(schedule)
             .withNotionals(faceAmount)
             .withCouponRates(coupons, accrualDayCounter)
+            .withFirstPeriodDayCounter(stubAccrualDayCounter==DayCounter() ? accrualDayCounter : stubAccrualDayCounter)
```

Comment:
Simply passing `stubAccrualDayCounter` will do the right thing (the null case is already managed in `FixedRateLeg`).

Reply:
That's great, I'll change and push.

Reply:
Pushed all above changes